### ISSUE TITLE
[GLPI10-RC1]Undefined index: id in C:\xampp\htdocs\glpi10\front\ticket.form.php at line 286

### ISSUE THREAD

**Issue by ukkt** [glpi-project/glpi#10684]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.0 RC1

### Bug description

Hello,

when I try to create a ticket, I get the error:

Undefined index: id in C:\xampp\htdocs\glpi10\front\ticket.form.php at line 286

### Relevant log output

```shell
[2022-02-17 07:02:25] glpiphplog.ALERT:   *** PHP Compile Error (64): Cannot declare class Glpi\Socket, because the name is already in use in C:\xampp\htdocs\glpi10\src\Socket.php at line 0  
[2022-02-17 07:02:32] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined index: id in C:\xampp\htdocs\glpi10\front\ticket.form.php at line 286  
[2022-02-17 07:02:32] glpiphplog.ALERT:   *** PHP Compile Error (64): Cannot declare class Glpi\Socket, because the name is already in use in C:\xampp\htdocs\glpi10\src\Socket.php at line 0  
[2022-02-17 07:02:45] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined index: id in C:\xampp\htdocs\glpi10\front\ticket.form.php at line 286  
[2022-02-17 07:02:45] glpiphplog.ALERT:   *** PHP Compile Error (64): Cannot declare class Glpi\Socket, because the name is already in use in C:\xampp\htdocs\glpi10\src\Socket.php at line 0  
[2022-02-17 07:02:48] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined index: id in C:\xampp\htdocs\glpi10\front\ticket.form.php at line 286  
[2022-02-17 07:02:48] glpiphplog.ALERT:   *** PHP Compile Error (64): Cannot declare class Glpi\Socket, because the name is already in use in C:\xampp\htdocs\glpi10\src\Socket.php at line 0  
[2022-02-17 07:02:49] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined index: id in C:\xampp\htdocs\glpi10\front\ticket.form.php at line 286  
[2022-02-17 07:02:49] glpiphplog.ALERT:   *** PHP Compile Error (64): Cannot declare class Glpi\Socket, because the name is already in use in C:\xampp\htdocs\glpi10\src\Socket.php at line 0  
[2022-02-17 07:02:50] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined index: id in C:\xampp\htdocs\glpi10\front\ticket.form.php at line 286  
[2022-02-17 07:02:50] glpiphplog.ALERT:   *** PHP Compile Error (64): Cannot declare class Glpi\Socket, because the name is already in use in C:\xampp\htdocs\glpi10\src\Socket.php at line 0  
[2022-02-17 07:02:51] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined index: id in C:\xampp\htdocs\glpi10\front\ticket.form.php at line 286  
[2022-02-17 07:02:51] glpiphplog.ALERT:   *** PHP Compile Error (64): Cannot declare class Glpi\Socket, because the name is already in use in C:\xampp\htdocs\glpi10\src\Socket.php at line 0  
[2022-02-17 07:02:53] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined index: id in C:\xampp\htdocs\glpi10\front\ticket.form.php at line 286  
[2022-02-17 07:02:53] glpiphplog.ALERT:   *** PHP Compile Error (64): Cannot declare class Glpi\Socket, because the name is already in use in C:\xampp\htdocs\glpi10\src\Socket.php at line 0
```

### Page URL

/front/ticket.form.php

### Steps To reproduce

1. Helpdesk
2. create Ticket

### Your GLPI setup information

<details><summary>Informationen über das System, die Installation und die Konfiguration</summary><pre>GLPI 10.0.0-rc1 (/glpi10 => C:\xampp\htdocs\glpi10)
Installation mode: TARBALL
Current language:de_DE

</details><details><summary>Server</summary><pre> 
Operating system: Windows NT xxx 6.2 build 9200 (Windows Server 2012 Datacenter Edition) i586
PHP 7.4.26 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apcu, bcmath, bz2,
	calendar, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imap, intl, json, ldap, libxml,
	mbstring, mysqli, mysqlnd, openssl, pcre, pdo_mysql, pdo_sqlite, readline, session, snmp, sodium, standard, tokenizer, xml,
	xmlreader, xmlrpc, xmlwriter, zip, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="2M" 
Software: Apache/2.4.52 (Win32) OpenSSL/1.1.1m PHP/7.4.26 (Apache/2.4.52 (Win32) OpenSSL/1.1.1m PHP/7.4.26 Server at xxx
	Port 80
)
	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36 Edg/98.0.1108.55
Server Software: MySQL Enterprise Server - Commercial
	Server Version: 8.0.11-commercial
	Server SQL Mode: STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION
	Parameters: root@xxx/glpi10
	Host info: XXXX via TCP/IP
	
PHP version (7.4.26) is supported.
Sessions configuration is OK.
Allocated memory is sufficient.
mysqli extension is installed.
Following extensions are installed: dom, fileinfo, json, simplexml.
curl extension is installed.
gd extension is installed.
intl extension is installed.
libxml extension is installed.
zlib extension is installed.
Database engine version (8.0.11) is supported.
The log file has been created successfully.
Write access to C:\xampp\htdocs\glpi10/files/_cache has been validated.
Write access to C:\xampp\htdocs\glpi10/config has been validated.
Write access to C:\xampp\htdocs\glpi10/files/_cron has been validated.
Write access to C:\xampp\htdocs\glpi10/files has been validated.
Write access to C:\xampp\htdocs\glpi10/files/_dumps has been validated.
Write access to C:\xampp\htdocs\glpi10/files/_graphs has been validated.
Write access to C:\xampp\htdocs\glpi10/files/_lock has been validated.
Write access to C:\xampp\htdocs\glpi10/files/_pictures has been validated.
Write access to C:\xampp\htdocs\glpi10/files/_plugins has been validated.
Write access to C:\xampp\htdocs\glpi10/files/_rss has been validated.
Write access to C:\xampp\htdocs\glpi10/files/_sessions has been validated.
Write access to C:\xampp\htdocs\glpi10/files/_tmp has been validated.
Write access to C:\xampp\htdocs\glpi10/files/_uploads has been validated.
Web access to files directory is protected
exif extension is installed.
ldap extension is installed.
openssl extension is installed.
zip extension is installed.
bz2 extension is installed.
Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to C:\xampp\htdocs\glpi10/marketplace has been validated.
Timezones seems not loaded, see https://glpi-install.readthedocs.io/en/latest/timezones.html.

</details><details><summary>GLPI constants</summary><pre> 
GLPI_ROOT: C:\xampp\htdocs\glpi10
GLPI_CONFIG_DIR: C:\xampp\htdocs\glpi10/config
GLPI_VAR_DIR: C:\xampp\htdocs\glpi10/files
GLPI_MARKETPLACE_DIR: C:\xampp\htdocs\glpi10/marketplace
GLPI_USE_CSRF_CHECK: 1
GLPI_CSRF_EXPIRES: 7200
GLPI_CSRF_MAX_TOKENS: 100
GLPI_USE_IDOR_CHECK: 1
GLPI_IDOR_EXPIRES: 7200
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: 
GLPI_TELEMETRY_URI: https://telemetry.glpi-project.org
GLPI_INSTALL_MODE: TARBALL
GLPI_NETWORK_MAIL: glpi@teclib.com
GLPI_NETWORK_SERVICES: https://services.glpi-network.com
GLPI_MARKETPLACE_ALLOW_OVERRIDE: 1
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: 1
GLPI_USER_AGENT_EXTRA_COMMENTS: 
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: 1
GLPI_AJAX_DASHBOARD: 1
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: 0
GLPI_CENTRAL_WARNINGS: 1
GLPI_DOC_DIR: C:\xampp\htdocs\glpi10/files
GLPI_CACHE_DIR: C:\xampp\htdocs\glpi10/files/_cache
GLPI_CRON_DIR: C:\xampp\htdocs\glpi10/files/_cron
GLPI_DUMP_DIR: C:\xampp\htdocs\glpi10/files/_dumps
GLPI_GRAPH_DIR: C:\xampp\htdocs\glpi10/files/_graphs
GLPI_LOCAL_I18N_DIR: C:\xampp\htdocs\glpi10/files/_locales
GLPI_LOCK_DIR: C:\xampp\htdocs\glpi10/files/_lock
GLPI_LOG_DIR: C:\xampp\htdocs\glpi10/files/_log
GLPI_PICTURE_DIR: C:\xampp\htdocs\glpi10/files/_pictures
GLPI_PLUGIN_DOC_DIR: C:\xampp\htdocs\glpi10/files/_plugins
GLPI_RSS_DIR: C:\xampp\htdocs\glpi10/files/_rss
GLPI_SESSION_DIR: C:\xampp\htdocs\glpi10/files/_sessions
GLPI_TMP_DIR: C:\xampp\htdocs\glpi10/files/_tmp
GLPI_UPLOAD_DIR: C:\xampp\htdocs\glpi10/files/_uploads
GLPI_INVENTORY_DIR: C:\xampp\htdocs\glpi10/files/_inventories
GLPI_NETWORK_REGISTRATION_API_URL: https://services.glpi-network.com/api/registration/
GLPI_MARKETPLACE_PLUGINS_API_URI: https://services.glpi-network.com/api/glpi-plugins/
GLPI_I18N_DIR: C:\xampp\htdocs\glpi10/locales
GLPI_VERSION: 10.0.0-rc1
GLPI_SCHEMA_VERSION: 10.0.0-rc1@4f12cc11183dfb486b84e28e8dad44a9259736c3
GLPI_MARKETPLACE_PRERELEASES: 1
GLPI_MIN_PHP: 7.4.0
GLPI_MAX_PHP: 8.2.0
GLPI_YEAR: 2022

</details><details><summary>Libraries</summary><pre> 
htmlawed/htmlawed version 1.2.6 in (C:\xampp\htdocs\glpi10\vendor\htmlawed\htmlawed)
phpmailer/phpmailer version 6.5.3 in (C:\xampp\htdocs\glpi10\vendor\phpmailer\phpmailer\src)
simplepie/simplepie version 1.5.8 in (C:\xampp\htdocs\glpi10\vendor\simplepie\simplepie\library)
mpdf/mpdf in (C:\xampp\htdocs\glpi10\vendor\mpdf\mpdf\src)
michelf/php-markdown in (C:\xampp\htdocs\glpi10\vendor\michelf\php-markdown\Michelf)
true/punycode in (C:\xampp\htdocs\glpi10\vendor\true\punycode\src)
iamcal/lib_autolink in (C:\xampp\htdocs\glpi10\vendor\iamcal\lib_autolink)
sabre/dav in (C:\xampp\htdocs\glpi10\vendor\sabre\dav\lib\DAV)
sabre/http in (C:\xampp\htdocs\glpi10\vendor\sabre\http\lib)
sabre/uri in (C:\xampp\htdocs\glpi10\vendor\sabre\uri\lib)
sabre/vobject in (C:\xampp\htdocs\glpi10\vendor\sabre\vobject\lib)
laminas/laminas-i18n in (C:\xampp\htdocs\glpi10\vendor\laminas\laminas-i18n\src)
laminas/laminas-servicemanager in (C:\xampp\htdocs\glpi10\vendor\laminas\laminas-servicemanager\src)
monolog/monolog in (C:\xampp\htdocs\glpi10\vendor\monolog\monolog\src\Monolog)
sebastian/diff in (C:\xampp\htdocs\glpi10\vendor\sebastian\diff\src)
donatj/phpuseragentparser in (C:\xampp\htdocs\glpi10\vendor\donatj\phpuseragentparser\src\UserAgent)
elvanto/litemoji in (C:\xampp\htdocs\glpi10\vendor\elvanto\litemoji\src)
symfony/console in (C:\xampp\htdocs\glpi10\vendor\symfony\console)
scssphp/scssphp in (C:\xampp\htdocs\glpi10\vendor\scssphp\scssphp\src)
laminas/laminas-mail in (C:\xampp\htdocs\glpi10\vendor\laminas\laminas-mail\src\Protocol)
laminas/laminas-mime in (C:\xampp\htdocs\glpi10\vendor\laminas\laminas-mime\src)
rlanvin/php-rrule in (C:\xampp\htdocs\glpi10\vendor\rlanvin\php-rrule\src)
blueimp/jquery-file-upload in (C:\xampp\htdocs\glpi10\vendor\blueimp\jquery-file-upload\server\php)
ramsey/uuid in (C:\xampp\htdocs\glpi10\vendor\ramsey\uuid\src)
psr/log in (C:\xampp\htdocs\glpi10\vendor\psr\log\Psr\Log)
psr/simple-cache in (C:\xampp\htdocs\glpi10\vendor\psr\simple-cache\src)
psr/cache in (C:\xampp\htdocs\glpi10\vendor\psr\cache\src)
league/csv in (C:\xampp\htdocs\glpi10\vendor\league\csv\src)
mexitek/phpcolors in (C:\xampp\htdocs\glpi10\vendor\mexitek\phpcolors\src\Mexitek\PHPColors)
guzzlehttp/guzzle in (C:\xampp\htdocs\glpi10\vendor\guzzlehttp\guzzle\src)
guzzlehttp/psr7 in (C:\xampp\htdocs\glpi10\vendor\guzzlehttp\psr7\src)
glpi-project/inventory_format in (C:\xampp\htdocs\glpi10\vendor\glpi-project\inventory_format\lib\php)
wapmorgan/unified-archive in (C:\xampp\htdocs\glpi10\vendor\wapmorgan\unified-archive\src)
paragonie/sodium_compat in (C:\xampp\htdocs\glpi10\vendor\paragonie\sodium_compat\src)
symfony/cache in (C:\xampp\htdocs\glpi10\vendor\symfony\cache)
html2text/html2text in (C:\xampp\htdocs\glpi10\vendor\html2text\html2text\src)
symfony/dom-crawler in (C:\xampp\htdocs\glpi10\vendor\symfony\dom-crawler)
twig/twig in (C:\xampp\htdocs\glpi10\vendor\twig\twig\src)
twig/string-extra in (C:\xampp\htdocs\glpi10\vendor\twig\string-extra)
symfony/polyfill-ctype not found
symfony/polyfill-iconv not found
symfony/polyfill-mbstring not found
symfony/polyfill-php80 in (C:\xampp\htdocs\glpi10\vendor\symfony\polyfill-php80)

</details><details><summary>SQL replicas</summary><pre> 
Not active

</details><details><summary>Notifications</summary><pre> 
Way of sending emails: SMTP (anonymous@srvexch)

</details><details><summary>Plugins list</summary><pre> 
	reports              Name: Berichte                       Version: 1.13.1     State: Installed / not activated
	order                Name: Bestellverwaltung              Version: 2.6.0      State: Installed / not activated
	datainjection        Name: Data injection                 Version: 2.9.0      State: Installed / not activated
	formcreator          Name: Form Creator                   Version: 2.10.1     State: Installed / not activated
	accounts             Name: Konten                         Version: 3.0.0-rc1  State: Enabled
	archires             Name: Network Architectures          Version: 2.7.0      State: Installed / not activated
	protocolsmanager     Name: Protocols manager              Version: 1.4        State: Enabled

</pre></details>

**Comment by SteadEXE** [comment]
Hello, it has been fixed here https://github.com/glpi-project/glpi/pull/10596
It will be probably shipped in the GLPI 10.0.0-rc2

**Comment by cconard96** [comment]
All listed errors in these logs should be fixed now.
Please test with the latest nightly "master" release.
https://nightly.glpi-project.org/glpi/
